project/AkkaBuild.scala

```diff
@@ -256,6 +258,11 @@ object AkkaBuild extends Build {
     )
```

Comment:
Just to keep it DRY you can `++ dontPublishSettings` to the end of the settings.